.astylerc

```diff
@@ -0,0 +1,57 @@
+### AStyle configuration options ###
+
+# Waterloo Rocketry does not enforce a team-wide style guide, but we do
+# ask that all codebases remain internally consistent. If you change
+# the options in this file, please rerun the formatter on all source
+# files using `./format.sh -a`
+
+# Documentation for AStyle and a description of what all options do
+# is at http://astyle.sourceforge.net/astyle.html.
+
+# Put opening braces on a new line after the expression, like this:
+#   if (x == 1)
+#   {
+style=kr
+indent=spaces=4
+
+# Indent case statements from parent switch
+indent-switches
+
+# In do-while loops, while appears on same line as the closing bracket:
+#   } while
+#   {
```

Comment:
I don't know why this opening brace is here

Reply:
removed; I forgot how do-while loops work